modules/learn/pages/clusters-and-availability/automatic-failover.adoc

```diff
@@ -108,10 +108,10 @@ This is illustrated by the following examples:
 | Node | Services Hosted
 
 | #1
-| Data
+| Data & Index
```

Comment:
I don't think including Index in this example is needed

Reply:
Removed & Index